WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -1009,12 +1006,9 @@ public void onStart() {
 
     @Override
     public void setTitle(@NonNull final String title) {
-        if (isAdded()) {
```

Comment:
Did you remove the `isAdded` check because we no longer need context when setting the title?

Reply:
I removed the `isAdded` check because I simplified this implementation using the `Me` fragment as reference – I have no idea whether it's necessary, so happy to take your lead here.

Reply:
I think it isn't necessary :-).